appium/options/transformers.py

```diff
@@ -0,0 +1,39 @@
+from datetime import timedelta
+from typing import Any, TypeVar, Generic, Type
+
+from appium.options.common.supports_capabilities import SupportsCapabilities
+
+T = TypeVar('T')
+C = TypeVar('C', bound='SupportsCapabilities')
+
+class OptionsDescriptor(Generic[T]):
```

Comment:
OptionsDescriptor class is some base entity. It must not be in the same module where transformers are defined

Reply:
Fixed.